docs/build-perf/issue-5218-incremental-check-results.md

```
@@ -0,0 +1,42 @@
+Raw command shapes:
+
+```bash
+touch crates/reinhardt-throttling/src/lib.rs && cargo check -p reinhardt-throttling
+touch crates/reinhardt-core/src/lib.rs && cargo check --workspace
+```
+
+## Interpretation
+
+The shared/core edit loop currently lands inside the expected 30-60% reduction
+range. The leaf package check does not: it is slower in this local run, so PR
+#5220 must not claim a universal incremental-check improvement.
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)